src/html.js

```diff
@@ -207,8 +207,8 @@ exports.html = function( texts ) {
             if ( k === 'italic' ) {            
                 span.style.fontStyle = obj[ k ] ? 'italic' : 'normal';
             }
-            if ( k === 'strikeout' ) {            
-                span.style.textDecoration = obj[ k ] ? 'line-through' : 'normal';
+            if ( k === 'strikeout' ) {
+                span.innerHTML = '<del>' + span.innerHTML + '</del>';          
```

Comment:
Can you check whether the carota parser can parse this properly? Please add a unit test if possible.